lib/content/commitlintrc.js

```diff
@@ -0,0 +1,11 @@
+// This file is automatically added by @npmcli/template-oss. Do not edit.
```

Comment:
Does this file need to be added to `.gitignore`?

Reply:
Same question for `.npmrc`

Reply:
No this file needs to be present in git so that there is a config to lint the commits with.

Reply:
Right, but our `.gitignore` ignores everything by default. What I meant to ask was does this need to be explicitly allowed in our gitignore with `!/.commitlintrc.js`.

Reply:
ooooh! yes! this PR is now yours, add it.